docs/table/performance.inc.rst

```diff
@@ -3,6 +3,8 @@
    from === to ***, the filename extension needs to be changed from .inc.rst to
    .rst, and a link needs to be added in the subpackage toctree
 
+.. doctest-skip-all
```

Comment:
Why skip all here? I thought the original intention was to test some of the stuff here, but maybe subpackage maintainers can confirm.

Reply:
Most of the examples here were excluded from tests individually anyways, and the couple that weren't did not contain any example output to be tested. That made me think that skipping all the tests wouldn't cause test coverage to drop much at all.